Practical_7/project_management.py

```diff
@@ -0,0 +1,16 @@
+import csv
+from project import Project
+from operator import attrgetter
+
+project_list = []
+with open('projects.txt', 'r', newline='') as data:
+    sorter = csv.reader(data, delimiter='\t')
+    data.readline()
+    for data in sorter:
```

Comment:
This is quite confused. Follow what we've taught.
Don't reuse variables in the same scope. Maybe you want the `data` file object again.